apps/nrm/translations/src/en/fields.json

```diff
@@ -333,6 +343,8 @@
     "bullet-2": "a formal decision – this can help if they’re being prosecuted for a crime they were forced to commit",
     "div-1-paragraph-1" : "We will need to share their personal details with organisations such as the police and the Home Office if they agree to their case being referred.",
     "div-1-paragraph-2": "Please explain to the potential victim the ways in which the Home Office may process their personal information, including the purposes for which it will be used, the legal basis and who it will be shared with. This is contained in the <a class=\"govuk-link\" href=\"https://www.gov.uk/government/publications/human-trafficking-victims-referral-and-assessment-forms/privacy-information-notice-national-referral-mechanism\">Privacy Information Notice</a> and <a class=\"govuk-link\" href=\"https://www.gov.uk/government/organisations/home-office/about/personal-information-charter\">Personal Information Charter.</a>",
+    "automatic-referral": "As the potential victim either under 18 years old, or their age is unknown, this case will automatically be referred.",
+    "automatic-referral-div-1-paragraph-1" : "We will need to share their personal details with organisations such as the police and the Home Office if their case is referred.",
```

Comment:
this all makes sense, do we need to run this by Richard and co to explain the changes including the content?

Reply:
yep all done

Reply:
and it was beautiful man 👍